pkg/frontend/v1/frontend_test.go

```diff
@@ -198,7 +199,7 @@ func TestFrontendCancel(t *testing.T) {
 	testFrontend(t, defaultFrontendConfig(), handler, test, nil, nil)
 }
 
-func TestFrontendMetricsCleanup(t *testing.T) {
+func TestFrontendMetrics(t *testing.T) {
```

Comment:
Why is the test name changing?

Reply:
Because this test is not just testing the `Cleanup` but also other parts of metrics.